### PR TITLE
Be able to lunch the upgrade throw make to never failed

### PR DESCRIPTION
**Pull request by sbrunner**
The related documentation will be added in the master branch

### CODE REVIEW

**Comment by elemoine** [comment]
Do you really mean to merge this in the 1.6 branch?

**Comment by sbrunner** [comment]
Yes because I want to add it in the master documentation and it will be easier it it's already available in the old (current) install

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -312,6 +312,7 @@ help:
 	@echo  "- update		Update the project (Git, Node, Pip)"
 	@echo  "- update-git-submodules	Update the Git submodules"
 	@echo  "- update-node-modules	Update node modules (using --force)"
+	@echo  "- upgrade		Lunch the upgrade process"
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Lunch => Launch

**Comment by asaunier** [comment]
No objection except that the commit message does not make much sense, you should reformulate it.

**Comment by asaunier** [comment]
throw => through

**Pull request merged**